Formula/luv.rb

```diff
@@ -0,0 +1,56 @@
+class Luv < Formula
+  desc "Bare libuv bindings for lua"
+  homepage "https://github.com/luvit/luv"
+  url "https://github.com/luvit/luv/archive/1.36.0-0.tar.gz"
+  version "1.36.0-0"
```

Comment:
Not blocking: I wonder if it would be worthwhile to add this version scheme to brew

Reply:
Are there enough formulae that use it to make it worthwhile?

Reply:
~17 formulae have a version string of a similar format: https://sourcegraph.com/search?q=repo:homebrew/homebrew-core+version%5C+%5C%22%5Cd%2B%28%5C.%5Cd%2B%29*-%5Cd%2B%5C%22&patternType=regexp

Reply:
Guess it might be worthwhile then!

Reply:
Question: should the version string `1.36-2` be older or newer than `1.36.2-0`?

Reply:
That is a good question. I don't know. `luv` seems to never leave it out [1], so it might be moot for this formula. However, it may matter elsewhere.

[1] https://github.com/luvit/luv/releases